app.py

```diff
@@ -1,36 +1,48 @@
 from flask import Flask, render_template, request, redirect, url_for
-import session_items as session
+import trello_items as item_store
+
+from dotenv import load_dotenv
+
+load_dotenv()
 
 app = Flask(__name__)
 app.config.from_object('flask_config.Config')
```

Comment:
You can remove this line and `flask_config.py` (as it's only used for the `SECRET_KEY` config value which is only used by the Flask session cookie)

Reply:
Removed